code/widgets/org.eclipse.scout.widgets.client/src/main/java/org/eclipse/scout/widgets/client/ui/forms/ChartFieldForm.java

```diff
@@ -1306,7 +1327,7 @@ protected ColorMode getConfiguredRef() {
               public class Auto extends AbstractMode<ColorMode> {
                 @Override
                 protected String getConfiguredText() {
-                  return "Auto";
+                  return "Element";
```

Comment:
Why is this called "Element" in the UI, wenn the corresponding technical mode is still called "Auto"?

Reply:
We use the ColorMode here as it has also 3 options and 2 of them match the ones we want to show. To be correct we need a second enum here with ELEMENT, DATASET and DATA